setup.py

```diff
@@ -1,21 +1,38 @@
 #!/usr/bin/env python
+import os
 import sys
 from setuptools import setup, find_packages
 from version import get_version
 from commands import preconfigure, configure, create_bucket_types, \
-    setup_security, enable_security, disable_security
+    setup_security, enable_security, disable_security, setup_timeseries
 
 install_requires = ['six >= 1.8.0']
 requires = ['six(>=1.8.0)']
 if sys.version_info < (2, 7, 9):
     install_requires.append("pyOpenSSL >= 0.14")
     requires.append("pyOpenSSL(>=0.14)")
-if sys.version_info < (3, ):
-    install_requires.append("riak_pb >=2.0.0")
-    requires.append("riak_pb(>=2.0.0)")
+
+riak_pb_in_pythonpath = False
+os_env_pythonpath = os.environ.get('PYTHONPATH')
+if os_env_pythonpath is not None:
+    for ppath in os_env_pythonpath.split(os.pathsep):
+        if ppath.find('riak_pb/python/lib') != -1:
+            riak_pb_messages = os.path.join(ppath, 'riak_pb', 'messages.py')
+            if os.path.exists(riak_pb_messages):
+                riak_pb_in_pythonpath = True
+                break
+
+if riak_pb_in_pythonpath:
```

Comment:
As you say, this will soon go away.  I guess I'd prefer just to manually manage your own dependencies for a manual install.  Seems like a temporary requirement.  Or we can rip this out as part of https://github.com/basho/riak-python-client/pull/418.